### PR TITLE
Feature/nlun 56 set up hosting

### PR DESCRIPTION
**Pull request by nilsml**
First version of dockerfile and radixconfig

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,42 @@
+FROM node:lts-alpine AS dependencies
+WORKDIR /opt/app
+COPY package.json package-lock.json ./
+RUN npm ci
```
Comment (wenche) [inline Dockerfile:4]: npm ci ❤️

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,42 @@
+FROM node:lts-alpine AS dependencies
```
Comment (wenche) [inline Dockerfile:1]: I guess lts is what we must do to be kind to future self, but let's just be aware of this 🕵️‍♀️

File: radixconfig.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    environmentConfig:
+    - environment: prod
+      resources:
+        requests:
+          memory: 512Mi
+          cpu: 100m
+        limits:
+          memory: 1Gi
+          cpu: 1000m
+      variables:
```
Comment (wenche) [inline radixconfig.yaml:23]: I guess this is correct, but it just looks funny with two variables following each other

**Review by wenche — APPROVED** [review]
LGTM 👍

File: radixconfig.yaml (hunk 3 above)
Comment (nilsml) [reply radixconfig.yaml:23]: Yeah, could omit them, but we will soon be using them

File: Dockerfile (hunk 2 above)
Comment (nilsml) [reply Dockerfile:1]: Yes. Maybe a little overkill right now, but still the file is quite easy to read and understand I think

**Pull request merged**